### DISCUSSION TITLE
Running Home Assistant container (core) install

### DISCUSSION THREAD

**Discussion by louie728** [SecKatie/ha-wyzeapi#237] [Q&A]
Hello all,

Im running HA on a rasberry pi 4 in docker.  I dont have HACS, supervisor mode etc...  Can I install this in some other manner?  I would really like to use my wyze equipment with HA.

TIA,

**Comment by SecKatie** [comment]
Yes,

HACS runs in docker on my machine.

Thanks,

Josh

**Comment by tggman** [comment]
I also don't have/use HACS (Home Assistant Community Store).  I instead simply download the zip file from https://github.com/JoshuaMulliken/ha-wyzeapi and copy the entire wyzeapi folder from the zip (located under in the custom_components folder in the zip file) and extract it directly into my HAC  config\custom_components folder.

**Reply by louie728** [reply]
Thanks guys :) I ended up installing HACS.
